inc/saiqueue.h

```diff
@@ -80,6 +80,15 @@ typedef enum _sai_queue_attr_t
      */
     SAI_QUEUE_ATTR_INDEX = 0x00000001,
 
+    /**
+     * @brief Parent scheduler node
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_SCHEDULER
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+     */
+    SAI_QUEUE_ATTR_PARENT_SCHEDULER_NODE = 0x00000002,
+
```

Comment:
In case of Hierarchical Qos not supported in some NPU's, queues are created directly on port. Then queue creation has attributes queue_index and port  oid. In you made it mandatory in those NPU's it again we have to give port oid as part it. which is redundant. So this attribute is mandatory in case hierarchy qos supported NPU's alone. else we can remove port oid and keep this one alone, So both case it will work.